app/src/mouse/pim447_trackball.c

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2021 Cedric VINCENT
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <drivers/sensor.h>
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(PIM447, CONFIG_SENSOR_LOG_LEVEL);
+
+/*
+ * It feels more natural and more confortable to square the speed
+ * reported by the PIM447 trackball.
+ */
+static int16_t convert_speed(int32_t value)
```

Comment:
Did you try plugging this into the existing acceleration code?

Reply:
No I didn't because I developed this driver independently from your branch initially.